### PR TITLE
Revert some of NextJS static path/props

### PR DESCRIPTION
**Pull request by tomquirk**
## What does this PR do and why?

Selectively reverting parts of https://github.com/jbx-protocol/juice-interface/pull/1555/ to solve https://github.com/jbx-protocol/juice-interface/issues/1634

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

File: src/middleware.page.ts [hunk 1]
```diff
@@ -1,43 +1,10 @@
 import { NextRequest, NextResponse } from 'next/server'
-import { paginateDepleteProjectsQueryCall } from 'utils/apollo'
 
-export async function middleware(request: NextRequest) {
+export function middleware(request: NextRequest) {
   // If request is for a handle id, add the search param with `isHandle`.
   if (request.nextUrl.pathname.startsWith('/@')) {
-    const handle = request.nextUrl.pathname.slice(2) // slice off "/@"
```
Comment (wraeth-eth) [inline src/middleware.page.ts:7]: ah that sucks that this is blowing up :(

File: src/pages/_app.page.tsx [hunk 2]
```diff
@@ -1,16 +1,103 @@
...
-import { Head } from 'components/common'
+import React, { useContext, useEffect, useState } from 'react'
+import { CoreAppWrapper, Head } from 'components/common'
+
+import { readNetwork } from 'constants/networks'
 
 import '../styles/antd.css'
 import '../styles/index.scss'
 
+// TODO: Move this to each page where needed.
```
Comment (wraeth-eth) [inline src/pages/_app.page.tsx:17]: I think moving this back will lose our SEO changes, but if it means site is reliable, probably a good call

Comment (tomquirk) [reply src/pages/_app.page.tsx:17]: Yep it will :( all g we'll get it back quick. Lots of the site is breaking atm tho

**Review by johnnyd-eth — APPROVED** [review]
Not able to trigger any of the runtime issues that have been popping up since #1555. Feel free to merge when that merge conflict is resolved

**Pull request merged**